examples/dune

```diff
@@ -1,7 +1,7 @@
 (executables
-  (names     bad simple)
+  (names     bad simple floats)
   (libraries alcotest))
 
 (alias
   (name runtest)
-  (deps bad.exe simple.exe))
+  (deps bad.exe simple.exe floats.exe))
```

Comment:
That's not really related to your changes, but I don't think that this runs the examples. Can you confirm that introducing a runtime error in the test is not caught by `dune runtest`?

Reply:
None of the examples are run on `dune runtest` as there is no action specified for runtest dune file in there.
This line could be added to run simple and floats: `(action (progn (run ./simple.exe) (run ./floats.exe)))`

Reply:
I'll fix that separately (the idiomatic way to do this is to turn the executable stanza into a test stanza)